.gitbook/5-oracles/1-operate-oracle-nodes/2-oracle-intialization.md

```diff
@@ -0,0 +1,144 @@
+# Initialization
+
+#### Before run the oracle, you should initialize the oracle configuration.
+
+## Preparations
+
+Before initializing the oracle, you should install the oracle described
+in [oracle-installation](./1-oracle-installation.md).
+After install the oracle, create an empty directory on your host, to be mounted as the `/home_mnt` directory in the
+enclave.
+
+```bash
+# If you run oracle on your host
+sudo mkdir /oracle
+
+# If you run oracle using Docker
+mkdir $(pwd)/oracle
+```
+
+After that, if you want to run the `oracled` with Docker as described in
+the [oracle-installation](./1-oracle-installation.md),
+it is recommended to create an environment variable that you can execute the Docker container easily.
+
+```bash
+export DOCKER_CMD="docker run --rm \
+  --device /dev/sgx_enclave \
+  --device /dev/sgx_provision \
+  -v $(pwd)/oracle:/oracle ghcr.io/medibloc/panacea-oracle:latest"
+```
+
+If you are not going to run the `oracled` without Docker, you can set an `DOCKER_CMD` environment variable empty string.
+
+```bash
+export DOCKER_CMD=""
```

Comment:
```suggestion
export DOCKER_CMD="ego run oracled"
```

Reply:
I applied the https://github.com/medibloc/panacea-core/pull/611#discussion_r1066637127 comment, so I deleted this part.